spec/controllers/users_controller_spec.rb

```diff
@@ -48,7 +48,7 @@
         expect(ActionMailer::Base.deliveries).not_to be_empty
       end
       it 'sends a confirmation mail containing the right message' do
-        expect(ActionMailer::Base.deliveries.last.body).to include(User.last.full_name)
+        expect(ActionMailer::Base.deliveries.last.body).to include(User.last.full_name.html_safe)
```

Comment:
The fabricator is generating something like "Peter O'Toole". This is the reason why the tests fail sometimes. See here:  https://circleci.com/gh/jepetko/myflix/30
This is actually no functional failure. It fails because the matcher is looking for "Peter O'Toole" in text like this:

```html
"<span> ...Peter O&#39;Toole. </span>"
```

This fixes the issue.